Defs/ThingDefs_Misc/Apparel_Headgear.xml

```diff
@@ -176,4 +176,123 @@
         </modExtensions>
     </ThingDef>
 
+    <ThingDef ParentName="ArmorHelmetMakeableBase">
+        <defName>Apparel_SimpleNightVision</defName>
+        <label>night vision monocular</label>
+        <description>A simple, monocular night vision device, worn strapped to the head.</description>
+        <graphicData>
+            <texPath>Things/Apparel/SimpleNightVision/SimpleNightVision_south</texPath>
+            <graphicClass>Graphic_Single</graphicClass>
+        </graphicData>
+    <techLevel>Industrial</techLevel>
+    <generateAllowChance>0.2</generateAllowChance>
+    <recipeMaker>
+      <unfinishedThingDef>UnfinishedTechArmor</unfinishedThingDef>
+      <skillRequirements>
+        <Crafting>6</Crafting>
+      </skillRequirements>
+        <researchPrerequisite>MicroelectronicsBasics</researchPrerequisite>      
+    </recipeMaker>
+    <costList>
+      <ComponentIndustrial>6</ComponentIndustrial>
+      <Steel>50</Steel>
+    </costList>
+        <statBases>
+            <WorkToMake>8000</WorkToMake>
+            <MaxHitPoints>100</MaxHitPoints>
+            <Mass>0.5</Mass>
+            <Bulk>3</Bulk>
+            <WornBulk>1</WornBulk>
+            <EquipDelay>0.5</EquipDelay>
+        </statBases>
+        <equippedStatOffsets>
+            <NightVisionEfficiency_Apparel>0.4</NightVisionEfficiency_Apparel>
+        </equippedStatOffsets>
```

Comment:
This won't work. it needs to go under statbases like this
```xml
<statBases> 
    <!-- . -->
    <!-- . -->
    <!-- At the end of StatBases -->
    <NightVisionEfficiency_Apparel>0.4</NightVisionEfficiency_Apparel>
</statBases> 
```

Reply:
Fixed.